src/elements/common/sub-header/Sort.js

```diff
@@ -42,7 +42,12 @@ const Sort = ({ sortBy, sortDirection, onSortChange }: Props) => (
                 const sortItemKey = `${sortByValue}${sortDirectionValue}`;
 
                 return (
-                    <MenuItem key={sortItemKey} onClick={() => onSortChange(sortByValue, sortDirectionValue)}>
+                    <MenuItem
+                        key={sortItemKey}
+                        isSelected={isSelected}
+                        isSelectItem
```

Comment:
this will update MenuItem's `role` to 'menuitemradio' and `aria-checked` to 'true'